submissions/SofiiaTrokhymchuk/pop-up-task/index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Pop-Up</title>
+</head>
+<body>
+    <div class="container">
+        <header class="header">
+            <img src="icons/kottans_logo.svg" alt="Logo" id="Logo">
+            <nav>
+                <ul class="header-menu">
+                    <li class="header-menu__item"><a href="#" class="header-menu__link">Home</a></li>
+                    <li class="header-menu__item"><a href="#" class="header-menu__link">About</a></li>
+                    <li class="header-menu__item"><a href="#" class="header-menu__link">Contact</a></li>
+                    <li class="header-menu__item">
+                        <input type="checkbox" id="pop-up-checkbox">
+                        <label for="pop-up-checkbox" id="pop-up-label">
+                            <div class="label-wrapper">
+                                <img src="icons/pop-up-icon.svg" alt="Pop-up icon" id="pop-up-icon">
+                            </div>         
+                        </label>  
+                        <div class="pop-up-wrapper">
+                            <div class="pop-up-container">
+                                <ul class="pop-up-menu">
+                                    <li class="pop-up-menu__item">
+                                        <a href="#">
+                                            <figure>
```

Comment:
I am not sure that `figure` tag is needed in this case.
Here is it's description. WDYT?

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/figure

Reply:
yep, I agree, that figure was an overhead. thank you!